lib/rules/no-fallthrough.js

```diff
@@ -39,6 +39,19 @@ function isReachable(segment) {
     return segment.reachable;
 }
 
+/**
+ * Checks whether a SwitchCase node is adjacent to the token after it
+ * @param {ASTNode} node - The case node to check
+ * @param {RuleContext} context - Rule context
+ * @returns {boolean} `true` if the case is adjacent to the next token
+ */
+function isAdjacentToNextToken(node, context) {
```

Comment:
Sorry I had missed this comment. I could come up with a good name since they could be on the same line, as mentioned below. What if I rewrite to `hasBlankLinesBetween(node, token)`?